Description: Do not call TestCase.setUp() twice
 The current test code works with testtools 0.9.35, but breaks hard with
 testtools 0.9.39, because TestCase.setUp() is called twice currently. This
 patch removes one of the 2 calls.
Author: Thomas Goirand <zigo@debian.org>
Forwarded: https://bugs.launchpad.net/python-keystoneclient/+bug/1371085
Last-Update: <YYYY-MM-DD>

--- python-keystoneclient-0.10.1.orig/keystoneclient/tests/v3/test_auth_saml2.py
+++ python-keystoneclient-0.10.1/keystoneclient/tests/v3/test_auth_saml2.py
@@ -48,7 +48,6 @@ class AuthenticateviaSAML2Tests(auth_uti
 
     def setUp(self):
         utils.TestCase.setUp(self)
-        auth_utils.TestCase.setUp(self)
 
         self.conf_fixture = auth_utils.TestCase.useFixture(self,
                                                            config.Config())
